### PR TITLE
Nept 1125

### PR DESCRIPTION
**Pull request by imanoleguskiza**

### CODE REVIEW

**Comment by voidtek** [comment]
:+1:

**Comment by voidtek** [comment]
:+1:

File: templates/menu_link/menu_link.component.inc [hunk 1]
```diff
@@ -6,6 +6,32 @@
...
+    }
+  }
+  // If the menu item has a submenu and has a depth of 1,
+  // render it as a button.
+  if (!empty($variables['sub_menu']) && $link['depth'] == 1) {
+    $variables['output']['#options']['attributes']['id'][] = 'nav-menu-expandable-label-' . $link['mlid'];
+    $variables['output']['#options']['attributes']['aria-controls'][] = 'nav-menu-expandable-group-' . $link['mlid'];
+    $variables['output']['#options']['attributes']['aria-expanded'][] = 'false';
+    $variables['output']['#options']['attributes']['aria-haspopup'][] = 'true';
+    $variables['output'] = '<button ' . drupal_attributes($variables['output']['#options']['attributes']) . '>' . $variables['output']['#text'] . '</button>';
```
Comment (drupol) [inline templates/menu_link/menu_link.component.inc]: As we are in a preprocess function, I would replace the HTML by the render array.

```
$variables['output'] = array(
  '#type' => 'html_tag',
  '#tag' => 'button',
  '...',
);
```

Comment (voidtek) [inline templates/menu_link/menu_link.component.inc]: Good improvement, I change that.

**Comment by voidtek** [comment]
:+1:

**Comment by voidtek** [comment]
:+1:

**Comment by drupol** [comment]
👍

**Pull request merged**